flamby/datasets/fed_camelyon16/dataset_creation_scripts/tiling_slides.py

```diff
@@ -20,12 +22,44 @@
 from flamby.utils import read_config, write_value_in_config
 
 
+def add_borders(slide: Slide, width: int = 10):
+    """Add borders to a slide thumbnail of the mean slide color.
+
+    This helps removing artefacts on the borders and prevents histolab TissuMask to
+    make incorrect detections.
+    """
+    pixels = slide.thumbnail.load()
+    pixel_values = np.zeros((slide.thumbnail.size[0], slide.thumbnail.size[1], 3))
+    for i in range(slide.thumbnail.size[0]):
+        for j in range(slide.thumbnail.size[1]):
+            pixel_values[i, j, :] = list(pixels[i, j])
```

Comment:
Are you sure you need a for-loop ? It is unclear to me the difference between pixels and `pixel_values`. Is pixels a weird data structure somehow ?

Reply:
yes it is a weird PIL stuff

Reply:
isn't there a way to call np.asarray() on it somehow ?